### PR TITLE
Update the blocker with updated design

### PR DESCRIPTION
**Pull request by jrosen081**
image

### CODE REVIEW

**Comment by stefan-philip** [comment]
This looks great! One note: how do we feel about the rectangular selection box around the star when it is being dragged? It may be a good accessibility feature? idk really just thought I'd bring it up.

**Review by stefan-philip — APPROVED** [review]

**Comment by jrosen081** [comment]
> This looks great! One note: how do we feel about the rectangular selection box around the star when it is being dragged? It may be a good accessibility feature? idk really just thought I'd bring it up.

Definitely a good point. I liked it for accessibility, but I will ask Stephanie her opinion!

File: frontend/src/components/Blocker.tsx [hunk 1]
```diff
@@ -1,60 +1,158 @@
...
+import background from "../assets/slider.svg";
 
 interface BlockerProps {
-    children: ReactElement,
-    afterSeconds: number,
-    message: string,
-    repeatable: boolean
+  children: ReactElement;
+  afterSeconds: number;
+  repeatable: boolean;
```
Comment (sarah-witt) [inline frontend/src/components/Blocker.tsx:17]: is there a reason why we need `repeatable`? it's always `false` for now.

**Review by sarah-witt — COMMENTED** [review]
Looks good and works well for me!

File: frontend/src/components/Blocker.tsx (hunk 1 above)
Comment (jrosen081) [reply frontend/src/components/Blocker.tsx:17]: That is a good point. I believe at one point there was a discussion of whether they should be repeated, and don't remember the ending to that conversation. I am going to leave it now and we can ask the researchers if that is something they would want at the next client meeting.

**Review by cindy1u0 — APPROVED** [review]
LGTM!

File: frontend/package.json [hunk 2]
```diff
@@ -11,10 +11,11 @@
     "@types/node": "^14.11.5",
     "@types/react": "^16.9.51",
     "@types/react-dom": "^16.9.8",
-    "antd": "^4.6.4",
+    "antd": "^4.7.0",
```
Comment (cindy1u0) [inline frontend/package.json:14]: it's prob fine but also we should make sure no other places break after we upgrade antd

**Review by sarah-witt — APPROVED** [review]

**Review by cindy1u0 — APPROVED** [review]

File: frontend/package.json (hunk 2 above)
Comment (jrosen081) [reply frontend/package.json:14]: Based on my testing nowhere is broken

**Comment by jrosen081** [comment]
@stefan-philip After  chatting with stephanie, I changed the color to sea foam and added a border radius!

**Pull request merged**